pkg/controllermanager/apis/config/v1alpha1/types.go

```diff
@@ -95,6 +98,18 @@ type ControllerRegistrationControllerConfiguration struct {
 	ConcurrentSyncs int `json:"concurrentSyncs"`
 }
 
+// EventControllerConfiguration defines the configuration of the
+// EventControllerConfiguration controller.
+type EventControllerConfiguration struct {
+	// ConcurrentSyncs is the number of workers used for the controller to work on
+	// events.
+	// +optional
+	ConcurrentSyncs int `json:"concurrentSyncs,omitempty"`
```

Comment:
```suggestion
	ConcurrentSyncs *int `json:"concurrentSyncs,omitempty"`
```

Optional fields are pointer types